addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3361,6 +3362,20 @@ const ImageHandlerOption = SnippetOptionWidget.extend({
      * @returns {Int} the maximum width at which the image can be displayed
      */
     _computeMaxDisplayWidth() {},
+    /**
+     * Adds background-image when image is optimizable.
+     *
+     * @private
+     * @abstract
+     */
+    _forceOptions(img) {},
```

Comment:
So name it `_applyImage` and review the comment: you should not speak about the background-image since it is not related to the Class you are documenting. Just say something like "Use the processed image where it needs to be used in the DOM", and document the parameter, below `@private`: `@param {HTMLImageElement} img`